tracer/tracer.go

```diff
@@ -133,7 +133,10 @@ func (t *Tracer) Flush() error {
 		return nil
 	}
 
-	_, err := t.transport.Send(spans)
+	// TODO[manu]: this doesn't make sense! change it with a proper buffer
```

Comment:
Example: breaking the logic (this can't work in a concurrent execution) doesn't break our tests